### DISCUSSION TITLE
如果设置早于开票时间的抢票时间，不会导致封号吗

### DISCUSSION THREAD

**Discussion by LII0146** [mikumifa/biliTickerBuy#103] [提问专区]
在之前的discussion中看到，建议将抢票时间设置成开票时间的前几分钟，例如20点开票，抢票时间设置成19:58。
但这样操作不是违背“不要抢没开票的票，否则会导致被封一段时间错过抢票”吗？
希望可以解释一下，谢谢

**Comment by mikumifa** [comment]
2点开票设2点，但是迟一点设

**Reply by LII0146** [reply]
迟一点设是什么意思呢

**Reply by YourComputer011** [reply]
意思应该是别让程序跑太久，提早个五六分钟差不多
以及我记着去年bw第一批大会员抢票刚一开始甚至无法选择票型，会提示“这个票种你买不了啊喂”，大约一分钟后才可以选择并提交，我一直不清楚这个会是b站特意做的风控机制，还是单纯bug。感觉抢票时间的设置见仁见智

**Reply by LII0146** [reply]
明白了，感谢
